DeutschJozsaAlgorithm/DeutschJozsaAlgorithm.ipynb

```diff
@@ -457,14 +463,25 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "open Microsoft.Quantum.Diagnostics;\n",
+    "// Start by implementing a function AllEqualityFactI \n",
```

Comment:
We can remove the part about defining a function altogether - we now have a library function `AllEqualityFactI` in `Microsoft.Quantum.Diagnostics` namespace, so we can recommend to use that one instead.